### ISSUE TITLE
detect host based on current context

### ISSUE THREAD

**Issue by mlbiam** [TremoloSecurity/kubectl-login#3]
based on the current context, run the login process instead of having to specify it as a parameter

**Comment by gregswift** [comment]
would be nice to not have to pass host if a context is already selected or exists. When you are already in a context this is handy when you are in right context cut logged out.

Current behaviors:

```
$ kubectl oulogin --host=login.context.url.com
```
or
```
$ kubectl ctx context-name
Switched to context "context-name".
$ kubectl get pods
Unable to connect to the server: failed to refresh token: oauth2: cannot fetch token: 401 Unauthorized
Response: 
$ kubectl oulogin --host=login.context.url.com
$ kubectl get pods
..works..
```

Would be nice to just do:
```
$ kubectl get pods
Unable to connect to the server: failed to refresh token: oauth2: cannot fetch token: 401 Unauthorized
Response: 
$ kubectl oulogin
```

**Comment by mlbiam** [comment]
fixed in 0.0.5.  Update by running `kubectl krew upgrade oulogin`
